### ISSUE TITLE
Use of Eval Causing Problems in Prettier-Atom (again)

### ISSUE THREAD

**Issue by kachkaev** [prettier/prettier#4764]
Looks like since prettier 1.13.0 the npm module contains calls to `eval()`, which do not work in Atom. This is the same issue as https://github.com/prettier/prettier/issues/1895 reported 1 year ago by @robwise (the main contributor to [prettier-atom](https://github.com/prettier/prettier-atom)).

Handling of `eval()` during build was removed in https://github.com/prettier/prettier/pull/4449 (https://github.com/prettier/prettier/commit/52a2a46b3c909320c738ef587095828dc20e21b0#diff-8ccb3827b1b22e3d261ea29ab2484e9eL60). I guess we need another https://github.com/prettier/prettier/pull/1925 to fix this! 😄

As far as I understand, [`eval("requre")`](https://github.com/prettier/prettier/blob/52a2a46b3c909320c738ef587095828dc20e21b0/src/main/parser.js#L48) is used to prevent Rollup from attempting to merge files where this should not be happening.

**Environments:**
- Prettier Version: 1.13.0...1.13.6
- Running Prettier via: Atom
- Runtime: Node 8
- Operating System: macOS

**Steps to reproduce:**

Install the latest prettier locally or globally, then install some prettier plugin, open atom and try formatting something.

**Expected behavior:**

No exception

**Actual behavior:**

```
/Users/me/.atom/packages/prettier-atom/dist/executePrettier/handleError.js:32 TypeError: eval(...) is not a function
    at /usr/local/lib/node_modules/prettier/index.js:26778:23
    at Array.map (<anonymous>)
    at loadPlugins (/usr/local/lib/node_modules/prettier/index.js:26775:121)
    at /usr/local/lib/node_modules/prettier/index.js:31113:16
    at Object.format (/usr/local/lib/node_modules/prettier/index.js:31134:12)
```

**Comment by duailibe** [comment]
Sorry about that!

**Comment by duailibe** [comment]
I published 1.13.7

> WDYT of adding a test for dist that would search for eval( and and fail if has been found?

Yeah I think we can add a test in the end of the build script / a step in the release script.

**Comment by j-f1** [comment]
@duailibe Don’t forget to [publish](https://github.com/prettier/prettier/releases/new) the link to the release notes.
